frontend/public/extend/devconsole/components/pipelines/PipelineVisualizationTask.scss

```
@@ -0,0 +1,118 @@
+$border-color: var(--pf-global--BorderColor--light);
+$gutter: 1.8em;
+
+.odc-pipeline__task {
+  position: relative;
+  width: 13em;
+  height: 2em;
+  white-space: normal;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  background-color: #fafafa;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  margin-top: 1.5em;
+  padding: 0.5em;
+
+  &.input-node {
+    width: 2em;
+    height: 2em;
+    border-radius: 2em;
+    color: var(--pf-global--BackgroundColor--200);
+    background: $border-color;
+    &::after {
+      content: '';
+      position: absolute;
+      transform: translateY(-100%);
+      left: 2em;
+      bottom: -1em;
+    }
+  }
+
+  &:not(:first-child) {
+    margin-top: 3em;
+  }
+
+  & .steps {
+    position: absolute;
+    top: -4px;
+    transform: translateY(-100%);
+  }
+
+  & .stepcount {
+    margin: 0 4px;
+    white-space: nowrap;
+  }
+
+  & .status {
```

Should be an element:
```suggestion
  &__status {
```